### PR TITLE
Automatically Triage New Metadata

### PR DESCRIPTION
**Pull request by wptfyibot**
PR for metadata triaged through /api/metadata/triage endpoint. See <insert a doc> for more information about how to use this service.

### CODE REVIEW

File: xhr/META.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+links:
+- product: chrome
+  url: crbug.com/571722
+  results:
+  - test: setrequestheader-header-allowed.htm
```
Comment (Hexcles) [inline xhr/META.yml:5]:
```suggestion
  - test: setrequestheader-header-allowed.htm
    subtest: XMLHttpRequest: setRequestHeader() - headers that are allowed (User-Agent)
```

File: xhr/META.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    subtest: XMLHttpRequest: setRequestHeader() - headers that are allowed (User-Agent)
```
Comment (Hexcles) [inline xhr/META.yml:6]:
```suggestion
    subtest: "XMLHttpRequest: setRequestHeader() - headers that are allowed (User-Agent)"
```

**Comment by stephenmcgruer** [comment]
@Hexcles should this be landed?

**Comment by Hexcles** [comment]
It should, and I think I did...

**Pull request merged**